scripts/eosio_build.sh

```diff
@@ -71,6 +71,8 @@ export BOOST_LINK_LOCATION=${OPT_LOCATION}/boost
 export LLVM_VERSION=release_40
 export LLVM_ROOT=${OPT_LOCATION}/llvm
 export LLVM_DIR=${LLVM_ROOT}/lib/cmake/llvm
+export CLANG8_ROOT=${OPT_LOCATION}/clang8
+export PINNED_COMPILER_VERSION=release_80
```

Comment:
I'm pretty sure this isn't a pin to a released version. The current compiler version I have after running the script is
`clang version 8.0.1 (https://git.llvm.org/git/clang.git a03da8be08a208122e292016cb6cea1f30229677)`